carto-package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "crankshaft",
+    "current_version": {
+        "requires": {
+            "postgres": ">=9.5",
```

Comment:
I think it's missing the python dependencies. In my local package I install first `python2 python2-joblib python2-pysal python2-scikit-learn python2-setuptools` but I'm not sure which ones are really required.

Reply:
the runtime dependencies are python, numpy and python-sklearn. I'll add them as well.